_studio/shared/include/cm_mem_copy.h

```diff
@@ -67,6 +67,11 @@ typedef mfxI32 cmStatus;
 #define min(a,b)            (((a) < (b)) ? (a) : (b))
 #endif
 
+static bool operator < (const mfxHDLPair & l, const mfxHDLPair & r)
```

Comment:
what is the idea of < operator ? How is it used?

Reply:
It is used to add pair to std::map in CmCopyWrapper::CreateCmSurface2D

Reply:
Still not clear to me. Oleg, do you see the point?

Reply:
std::map keys should be comparable using operator <. Still I'd request to add explicit comment in the code to reduce confusion.

Reply:
I see now.  Thanks. Yes, Maxim, please add a comment.

Reply:
Done.